src/coloring/contraction_coloring.jl

```diff
@@ -80,17 +77,16 @@ function max_degree_vertex(G::VSafeGraph)
         end
     end
     return max_degree_vertex
-
 end
 
 
 """
-    non_neighbors(G,x)
+    non_neighbors(G, x)
 
 Find the set of vertices belonging to the graph G which do
 not share an edge with the vertex x.
 """
-function non_neighbors(G::VSafeGraph, x::Int)
+function non_neighbors(G::VSafeGraph, x::Integer)
```

Comment:
```suggestion
function non_neighbors(G::VSafeGraph, x::Int)
```
Reverting changes since this is one of the rare ones to function signatures